shared/src/packages/sections/libs/types/section-get-all-item-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type SectionGetAllItemResponseDto = {
+  id: number;
+  name: string;
+  content: string;
```

Comment:
```suggestion
type SectionGetAllItemResponseDto<T = unknown> = {
  id: number;
  name: string;
  content: T;
```
since this is a json